test/functional/interface_rest.py

```diff
@@ -103,6 +103,14 @@ def run_test(self):
         n, = filter_output_indices_by_value(json_obj['vout'], Decimal('0.1'))
         spending = (txid, n)
 
+        # Test /tx with an invalid and an unknown txid
+        invalid_param = "abc"
+        unknown_param = "0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Ah, nice that these can be reused.  Since the reuses are much further down, maybe hoist these to constants at the top.

Reply:
Cool! Done!